databases/business-sync/fragment-shoebox-item.js

```diff
@@ -62,6 +62,74 @@
           }
         }
       }
+    },
+    annotations: {
+      type: 'array',
+      required: false,
+      arrayElementsValidator: {
+        type: 'object',
+        required: true,
+        propertyValidators: {
+          // indicates what type of annotation this is
+          // Only embedded is supported to start
+          type: {
+            type: 'enum',
+            immutable: true,
+            required: true,
+            predefinedValues: [ 'embedded' ]
+          },
+          // a simple tag describing what type data this annotation is
+          // example:  'user-metadata', 'ocr-analysis', 'model-XYZ-classification'
+          dataType: {
+            type: 'enum',
+            immutable: true,
+            required: true,
+            predefinedValues: [ 'user-metadata' ]
+          },
+          // payload, pretty much unrestricted at this point
+          data: {
+            type: 'object',
+            required: true
+          },
+          // Array of timestamp and user IDs of modifications to this annotation
+          modifications: {
+            type: 'array',
+            mustNotBeEmpty: true,
+            required: true,
+            arrayElementsValidator: {
+              type: 'object',
+              propertyValidators: {
+                userId: {
```

Comment:
in the case of annotations that may be added through automated processes, what would we expect the `userId` to be?  would requiring a Books user ID still be appropriate in all cases?